src/main/java/name/valery1707/kaitai/KaitaiMojo.java

```diff
@@ -167,9 +173,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 				.generator(kaitai, output, packageName)
 				.withSource(source)
 				.overwrite(overwrite)
-				.generate(getLog());
+				.generate(logger);
 		} catch (KaitaiException e) {
-			throw new MojoExecutionException(
+			throw new KaitaiException(
```

Comment:
\[Checkstyle\] WARNING: 'catch' child has incorrect indentation level 24, expected level should be 6\.